src/components/Carousel/Carousel.vue

```diff
@@ -1,35 +1,86 @@
 <template>
-  <div>
-    <carousel
-      :perPageCustom="[[320, 2], [640, 4]]"
-      :spacePadding="10"
-      :mouse-drag="true"
-      :loop="true"
-      :centerMode="false"
-      :scrollPerPage="false"
-    >
-      <slot />
-    </carousel>
-  </div>
+  <carousel
+    :class="getClass"
+    :perPageCustom="[[320, 1], [480, 2], [640, 4]]"
+    :spacePadding="0"
+    :mouse-drag="true"
+    :loop="true"
+    :centerMode="false"
+    :scrollPerPage="false"
+    :navigationEnabled="sideNav"
+    :paginationEnabled="getNavEnabled"
+    :paginationPosition="getNavPosition"
+    :paginationColor="getNavColor"
+    :paginationActiveColor="getNavActiveColor"
+  >
+    <slot />
+  </carousel>
 </template>
 
 <script>
 import { Carousel } from "vue-carousel";
-import Slide from "../Slide/Slide.vue";
 
 export default {
   // This is named as such to avoid collision with the existing <carousel />
   name: "CarouselComponent",
-  components: { Carousel, Slide },
+  components: { Carousel },
+  props: {
+    dotNav: Boolean,
+    barNav: Boolean,
+    sideNav: Boolean
+  },
+  created: function() {
+    console.table({
+      dotNav: this.dotNav,
+      barNav: this.barNav
+    });
+  },
+  computed: {
+    getNavEnabled: function() {
```

Comment:
prefer not to have "get" on computed properties. Note that computed properties can technically have setters and getters